src/App.js

```diff
@@ -1,25 +1,62 @@
 import React from 'react';
+import { Clock } from './components/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+export class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    name: '0',
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  hideClock = () => {
+    this.setState({
+      isClockVisible: false,
+    });
+  }
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+  showClock = () => {
+    this.setState({
+      isClockVisible: true,
+    });
+  }
 
-export default App;
+  changeName = () => {
+    this.setState({
+      name: (Math.random() * 100).toFixed(0),
+    });
+  }
+
+  render() {
+    const { name } = this.state;
+
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        {this.state.isClockVisible && <Clock name={name} />}
+        <div className="App__buttons">
+          <button
+            type="button"
+            onClick={this.showClock}
+          >
+            Show
+          </button>
+          {' '}
```

Comment:
```suggestion
          {' '}
```
You could use some css to create margins